lib/studies/reply/app.dart

```diff
@@ -38,6 +44,35 @@ class ReplyApp extends StatelessWidget {
         routes: <String, WidgetBuilder>{
           homeRoute: (context) => const AdaptiveNav(),
         },
+        onGenerateRoute: (settings) {
+          switch (settings.name) {
+            case homeRoute:
+              return MaterialPageRoute<void>(
+                builder: (context) {
+                  return const AdaptiveNav();
+                },
+              );
+              break;
+            case searchRoute:
+              return PageRouteBuilder<void>(
+                pageBuilder: (context, animation, secondaryAnimation) {
```

Comment:
nit: =>

Reply:
This spanned more than one line so I used brackets instead.